src/PKSim.Core/Model/AdvancedParameterFactory.cs

```diff
@@ -56,16 +61,14 @@ public AdvancedParameter Create(IParameter parameter, DistributionType distribut
             distributionMetaData.MaxValue = parameter.MaxValue;
             distributionMetaData.MaxIsAllowed = parameter.MaxIsAllowed;
          }
-         distributionMetaData.Dimension = parameter.Dimension.Name;
-         distributionMetaData.DefaultUnit = parameter.DisplayUnit.Name;
 
-         distributionMetaData.ParameterName = "distribution";
          advancedParameter.DistributedParameter = _parameterFactory.CreateFor(distributionMetaData);
 
          //should set the distribution parameter as visible since they will need to be edited
          advancedParameter.DistributedParameter.AllParameters().Each(x =>
          {
             x.Visible = true;
+            x.IsDefault = false;
```

Comment:
This was probably missing here too. Those parameter are definitly input

Reply:
Yep